packages/nuxt/src/core/nuxt.ts

```diff
@@ -557,6 +557,12 @@ async function initNuxt (nuxt: Nuxt) {
     addPlugin(resolve(nuxt.options.appDir, 'plugins/payload.client'))
   }
 
+  // Show compatibility version banner when Nuxt is running with a compatibility version
+  // that is different from the current major version
+  if (!(satisfies(nuxt._version, nuxt.options.future.compatibilityVersion + '.x'))) {
+    console.info(`running with compatibility version \`${nuxt.options.future.compatibilityVersion}\``)
```

Comment:
```suggestion
    console.info(`Running with compatibility version \`${nuxt.options.future.compatibilityVersion}\``)
```

Would probably capitalize this - otherwise LGTM 👍🏻